src/toast/stories/ToastBasic.component.tsx

```diff
@@ -25,7 +26,6 @@ export default App;
 
 const Notifications = () => {
   const { toasts, startPause, endPause, removeToast } = useToasts();
```

Comment:
Maybe we can name these 

pauseTimer & resumeTimer ??? feels more natural. 🤔

Reply:
yeah, more descriptive about what it does.